common/src/chain/transaction.rs

```diff
@@ -104,38 +104,3 @@ impl Transaction {
         }
     }
 }
-
```

Comment:
@iljakuklic should we drop this test? Just double-checking.

Reply:
I'd drop it. The maintenance burden is going to greatly outweigh its usefulness in my view/